packages/rancher-logging/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -23,11 +23,32 @@
 +{{- end }}
 +{{- end }}
 +
-+{{- define "windowsPathPrefix" }}
-+{{- $temp := (default "c:\\" .Values.global.cattle.rkeWindowsPathPrefix | replace "\\" "/" | replace "//" "/") }}
-+{{- if (hasSuffix "/" $temp) }}
-+{{- trimSuffix "/" $temp }}
-+{{- else }}
-+{{- $temp }}
-+{{- end }}
-+{{- end }}
++{{- define "windowsPathPrefix" -}}
++{{- default "c:\\" .Values.global.cattle.rkeWindowsPathPrefix | replace "\\" "/" | replace "//" "/" | replace "c:" "C:" -}}
++{{- end -}}
++
++{{- define "windowsInputTail" -}}
++{{- $temp := (include "windowsPathPrefix" .) -}}
++{{- if (hasSuffix "/" $temp) -}}
++{{- trimSuffix "/" $temp -}}
++{{- else -}}
++{{- $temp -}}
++{{- end -}}
++{{- end -}}
++
++{{- define "windowsKubernetesFilter" -}}
++{{- $temp := ((include "windowsPathPrefix" .) | replace ":" "" | replace "/" ".") -}}
++{{- if (hasSuffix "." $temp) -}}
++{{- $temp = trimSuffix "." $temp -}}
++{{- end -}}
++{{- printf "kubernetes.%s" $temp -}}
++{{- end -}}
++
++{{- define "windowsInputTailMount" -}}
++{{- $temp := ((include "windowsPathPrefix" .) | replace "C:" "") -}}
++{{- if (hasSuffix "/" $temp) -}}
++{{- trimSuffix "/" $temp -}}
++{{- else -}}
++{{- $temp -}}
++{{- end -}}
++{{- end -}}
```

Comment:
**Note to reviewers:** Due to the Kubebuilder spec, these must start with /. This cannot use the windowsInputTail as a result. This has been tested on a Windows 20H2 cluster on k8s 1.20.6 with RKE.

Source: https://github.com/banzaicloud/logging-operator/blob/f18b277cbdb8b4dd4db3ac15f8dd8f50c3e28df5/pkg/sdk/api/v1beta1/node_agent_types.go#L59